### PR TITLE
Add api service exception for permissions

### PR DESCRIPTION
**Pull request by bigglesandginger**
Allow for api-service calls to bypass permissions (assume they were spawned by a valid user)

### CODE REVIEW

**Review by Tom-Szendrey — APPROVED** [review]
I have only tested this with a simulation with pyciemss but this has helped resolve that issue thank you!!

File: packages/server/src/main/resources/application.properties [hunk 1]
```diff
@@ -63,6 +63,8 @@ spring.security.oauth2.client.provider.keycloak.issuer-uri=${terarium.keycloak.u
 spring.security.oauth2.client.registration.keycloak.client-id=${terarium.keycloak.client-id:app}
 spring.security.oauth2.resourceserver.jwt.issuer-uri=${terarium.keycloak.url:localhost}/realms/${terarium.keycloak.realm:Terarium}
 
+terarium.keycloak.api-service-name=api-service
```
Comment (dvince2) [inline packages/server/src/main/resources/application.properties:66]: Is there any risk having this username here in plain text?

**Review by dvince2 — APPROVED** [review]

File: packages/server/src/main/resources/application.properties (hunk 1 above)
Comment (bigglesandginger) [reply packages/server/src/main/resources/application.properties:66]: I don't think so.  The password is in the encrypted blobs in orchestration, and this is for developer running - again orchestration can override this with whatever we deem should be the user name,  though I for one don't think of the user name being a secret, just the password... thoughts?

Comment (dvince2) [reply packages/server/src/main/resources/application.properties:66]: ah! no that makes sense. just wanted to double check.

**Pull request merged**